.delivery/build_cookbook/libraries/helpers.rb

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright:: Copyright 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'aws-sdk'
+require 'json'
+require 'open-uri'
+
+module BuildCookbook
+  module Helpers
+    #########################################################################
+    # Github
+    #########################################################################
+    def skip_omnibus_build?
+      pr_has_label?(node['build-cookbook']['pr_labels']['omnibus_skip_build'])
+    end
+
+    #########################################################################
+    # AWS
+    #########################################################################
+    def aws_profile
+      node['build-cookbook']['aws_profile']
+    end
+
+    def aws_key_name
+      node['build-cookbook']['aws_key_name']
+    end
+
+    def aws_private_key
+      node['build-cookbook']['aws_private_key']
+    end
```

Comment:
These AWS helpers won't be needed after https://github.com/chef-cookbooks/expeditor-build/pull/23 is merged.